runtime-modules/working-group/src/lib.rs

```diff
@@ -1329,13 +1365,28 @@ impl<T: Trait<I>, I: Instance> Module<T, I> {
             )
             .map_err(|e| DispatchError::Other(e))
     }
+
+    fn ensure_worker_role_storage_text_is_valid(text: &[u8]) -> DispatchResult {
+        Self::worker_storage_size()
+            .ensure_valid(
+                text.len(),
+                Error::<T, I>::WorkerStorageTextTooShort.into(),
+                Error::<T, I>::WorkerStorageTextTooLong.into(),
+            )
+            .map_err(|e| DispatchError::Other(e))
+    }
 }
 
 /// Creates default text constraint.
 pub fn default_text_constraint() -> InputValidationLengthConstraint {
     InputValidationLengthConstraint::new(1, 1024)
 }
 
+/// Creates default storage size constraint.
```

Comment:
Thinking though this again, setting default to allow 0 min is okay, but it means in future the lead can set different constraint with non-zero min. If that is a policy then okay, but if we want to always allow 0 size
we can drop `InputValidationLengthConstraint` and just have the upper bound configurable.

Reply:
done https://github.com/Joystream/joystream/pull/2180/commits/33f53545ed1b9cff1ceb7adc3a0b4a705766ce68